HerPublicWebsite.BusinessLogic/Services/QuestionFlow/QuestionFlowService.cs

```diff
@@ -207,5 +208,17 @@ private QuestionFlowStep HouseholdIncomeForwardDestination(Questionnaire questio
         {
             return QuestionFlowStep.CheckAnswers;
         }
+
+        private QuestionFlowStep CheckAnswersForwardDestination(Questionnaire questionnaire)
+        {
+            return (questionnaire.IncomeBand, questionnaire.HasGasBoiler, questionnaire.FoundEpcBandIsTooHigh, questionnaire.EpcDetailsAreCorrect, questionnaire.Country, questionnaire.OwnershipStatus, questionnaire.IsLsoaProperty) switch
+            {
+                (IncomeBand.UnderOrEqualTo31000, not HasGasBoiler.Yes, false, _, Country.England, OwnershipStatus.OwnerOccupancy, _) => QuestionFlowStep.Eligible,
+                (IncomeBand.UnderOrEqualTo31000, not HasGasBoiler.Yes, _, false, Country.England, OwnershipStatus.OwnerOccupancy, _) => QuestionFlowStep.Eligible,
+                (IncomeBand.GreaterThan31000, not HasGasBoiler.Yes, false, _, Country.England, OwnershipStatus.OwnerOccupancy, true) => QuestionFlowStep.Eligible,
+                (IncomeBand.GreaterThan31000, not HasGasBoiler.Yes, _, false, Country.England, OwnershipStatus.OwnerOccupancy, true) => QuestionFlowStep.Eligible,
+                _ => QuestionFlowStep.Ineligible,
```

Comment:
Can we link to the eligibility rules in any way here, so there's a clear documentation of the policy we're trying to implement?

Reply:
Also please move this logic to a calculated property on the Questionnaire class. This is the core logic of the whole service, so it shouldn't be hidden here.

Reply:
Yes, that seems like a good idea